src/dns.py

```diff
@@ -482,11 +488,15 @@ def _get_dns_zone_for_domain(domain):
         zone = parent_list[-1]
 
     logger.warning(
-        f"Could not identify the dns zone for domain {domain}, returning {zone}"
+        f"Could not identify correctly the dns zone for domain {domain}, returning {zone}"
     )
     return zone
 
 
+def _get_relative_name_for_dns_zone(domain, base_dns_zone):
+    return re.sub("\.?" + base_dns_zone.replace(".", "\.") + "$", "", domain.strip(".")) or "@"
```

Comment:
```
src/dns.py:497:20: W605 invalid escape sequence '\.'
src/dns.py:497:55: W605 invalid escape sequence '\.'
```

Uuuh not sure why the linter is unhappy about that

Reply:
```suggestion
    return re.sub(r"\.?" + base_dns_zone.replace(".", r"\.") + "$", "", domain.strip(".")) or "@"
```